packages/cli/src/util/build/scrub-argv.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Redacts `--env`, `--build-env`, and `--token` in `argv`.
+ */
+export function scrubArgv(argv: string[]) {
+  const clonedArgv = [...argv];
+  const tokenRE = /^(-[A-Za-z]*[bet]|--(?:build-env|env|token))(=[^\w]*)?/;
```

Comment:
I'm confused why this is using a regex here. I thought we already parsed arvg using [`arg`](https://www.npmjs.com/package/arg) package. Could we reuse the parsed result instead?

Reply:
The original code was logging `process.argv`. My intent was to preserve this and not output the parsed `args`.

Reply:
Although if we moved to using `client.args` and stringified it, it would only print any parsed args. `--token` is a no-op for build, so would just get dropped.